templates/products/inventory.php

```diff
@@ -1,3 +1,8 @@
+<?php
+// Get product stock quantity by postid
+$product = wc_get_product( $post_id );
```

Comment:
Pls change to `$product = wc_get_product( $post_id ?? 0 );` if `$post_id` not defined.